rpm/tarantool.spec

```diff
@@ -207,11 +208,7 @@ make %{?_smp_mflags}
 rm -rf %{buildroot}%{_datarootdir}/doc/tarantool/
 
 %check
-# Blocked testing starting from Fedora 33 while Python 3 not enabled
-# for testing, issue for switching it on is tarantool/tarantool-qa#17.
```

Comment:
You may want to close the issue right from the commit message:

```
Closes tarantool/tarantool-qa#17
```

Reply:
added to commit message